package.json

```diff
@@ -1,12 +1,12 @@
 {
   "name": "@suffolklitlab/alkiln",
-  "version": "4.10.1-pre-cli",
+  "version": "4.11.0",
   "description": "Integrated automated end-to-end testing with docassemble, puppeteer, and cucumber.",
   "main": "lib/index.js",
   "scripts": {
     "test": "npm run unit && npm run langs_setup && npm run cucumber -- $@",
     "cucumber": "./lib/run_cucumber.js",
-    "langs": "npm run langs_setup && ./node_modules/.bin/cucumber-js --require ./lib/index.js docassemble/*/data/sources/*_lang*.feature",
+    "langs": "npm run langs_setup && npm run cucumber",
```

Comment:
Can we avoid adjusting this until we actually start messing with the language features? We haven't for a while, so I'm not even sure they work anymore. From what I can tell/recall this is supposed to run just language feature files and I don't think the above implements that limit.

Reply:
As it was, it would break with this refactor, so I changed it to at least work. I was going to mention that when this gets out of draft. 

I know that it doesn't run just the language feature files; if we want to support that, I can change things, but I'm not going to leave outdated code that is known to be broken around.

Reply:
Good point. In that case, maybe we remove it for now and revisit when we have time to dedicate to it? Maybe less confusing than leaving a remnant whose purpose is unclear?

Reply:
I can remove it, but that breaks compatability, so it's a major version change.

IMO the sctript is still useful: it's running a strict superset of the language tests, so it's still running all of the tests that script would run if it was perfect. If your normal tests pass, then we're just doing duplicate work (and if the normal tests don't pass, the other scripts would fail too). 

I acknowledge that it's unclear when reading the code, I can add a comment to avoid that. And if this holds up the feature, I can fix it by changing the arguments to run_cucumber.js. Now is the time to do that if we want.

It feels like the core of the issue is what to do with langs as a feature. It might be worth a discussion before this is merged

Reply:
Let's just make an issue to get langs to only run lang files. For now if it runs everything, it'll be longer, but at least it'll still run langs.

Reply:
Made a quick issue for this: https://github.com/SuffolkLITLab/ALKiln/issues/653